src/classes/OPP_AutomatedSoftCreditsService.cls

```diff
@@ -38,6 +38,10 @@
 public class OPP_AutomatedSoftCreditsService {
 
     private static final String RELATED_OPPORTUNITY_CONTACT_ROLE_NONE = '--None--';
+    private static final String OBJECT_TYPE_RELATIONSHIP = 'npe4__Relationship__c';
+    private static final String OBJECT_TYPE_AFFILIATION = 'npe5__Affiliation__c';
+    private static final String FILED_NAME_RELATED_CONTACT = 'npe4__RelatedContact__c';
```

Comment:
**Field** is misspelled as `FILED` in these constants.

Reply:
Good catch.